File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/mrqc/i/ICiMrQcParamService.java
package iih.ci.mrqc.i;

import xap.mw.core.data.BizException;

public interface ICiMrQcParamService {

	/**
	 * 获取门诊病历召回最大申请天数
	 * @param id_org
	 * @return
	 * @throws BizException
	 */
	public abstract String GetSysParamOpRcMaxDays(String id_org) throws BizException;
	
	/**
	 * 
	 * @param id_org
	 * @return 门诊病历召回默认时间
	 * @throws BizException
	 */
	public abstract String GetSysParamOpRcDefaultDays(String id_org) throws BizException;
}

File: client/iih_stub/iih.ci_stub/iih.ci.mrqc/i/ICiMrQcParamService.cs
using System;

namespace iih.ci.mrqc.i
{
    public interface ICiMrQcParamService
    {
        /// <summary>
        /// 获取门诊病历召回最大申请天数
        /// </summary>
        /// <param name="id_org"></param>
        /// <returns></returns>
        string GetSysParamOpRcMaxDays(String id_org);

     /**
	 * 
	 * @param id_org
	 * @return 门诊病历召回默认时间
	 * @throws BizException
	 */
	string GetSysParamOpRcDefaultDays(String id_org) ;
    }
}
